### PR TITLE
feat(router): 添加武汉纺织大学就业信息

### PR DESCRIPTION
**Pull request by orion-tan**
## 该 PR 相关 Issue / Involved Issue

Close #

## 路由地址示例 / Example for the Proposed Route(s)

```routes
/wtu/job/xxtz
/wtu/job/tzgg
/wtu/job/xwkd
```

## 新 RSS 路由检查表 / New RSS Route Checklist
  
- [X] 新的路由 New Route
  - [X] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [X] 文档说明 Documentation
  - [X] 中文文档 CN
  - [ ] 英文文档 EN
- [X] 全文获取 fulltext
  - [X] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [X] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [X] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -116,6 +116,7 @@
     "markdown-it": "13.0.1",
     "module-alias": "2.2.2",
     "notion-to-md": "3.0.1",
+    "pako": "2.1.0",
```
Comment (TonyRL) [inline package.json:119]: No need to install `pako`, you can use built-in `zlib` to achieve the same thing.

File: lib/v2/wtu/job.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { URL } = require('url');
```
Comment (TonyRL) [inline lib/v2/wtu/job.js:3]:
```suggestion
```
No need to require global object.
docs: https://nodejs.org/api/url.html#class-url

File: package.json (hunk 1 above)
Comment (ticks-tan) [reply package.json]: Okay, I have made the modifications.

File: lib/v2/wtu/radar.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    '91wllm.com': {
+        _name: '武汉纺织大学',
+        wtu: [
+            {
+                title: '就业信息',
+                docs: 'https://docs.rsshub.app/university.html#wtu-job',
```
Comment (TonyRL) [inline lib/v2/wtu/radar.js:7]:
```suggestion
                docs: 'https://docs.rsshub.app/university.html#wu-han-fang-zhi-da-xue',
```

**Pull request merged**